default.nix

```diff
@@ -2,7 +2,7 @@ let requiredVersion = import ./lib/minver.nix; in
 
 if ! builtins ? nixVersion || builtins.compareVersions requiredVersion builtins.nixVersion == 1 then
 
-  abort ''
+  throw ''
```

Comment:
Is this maybe a special case? Some commands like `nix-env -qa` will silently ignore `throw`.